src/main/java/RacingGame.java

```diff
@@ -0,0 +1,32 @@
+import domain.Car;
+import view.ResultView;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class RacingGame {
+
+    private static final Integer CONDITION = 4;
+    Random random = new Random();
+
+    public List<Car> makeCars(int carCnt) {
+        List<Car> cars = new ArrayList<Car>(carCnt);
```

Comment:
변수 타입을 List<Car>로 선언하면 List가 Car객체를 갖는 것으로 추측이 가능해서 ArrayList에 객체타입을 명시안해도 됩니다!
그리고 리스트 사이즈도 생략이 가능합니다.
```suggestion
        List<Car> cars = new ArrayList<>();
```